src/main/java/rx/plugins/RxJavaPlugins.java

```diff
@@ -149,7 +164,27 @@ private static Object getPluginImplementationViaProperty(Class<?> pluginClass) {
          * This will only happen during system startup thus it's okay to use the synchronized
          * System.getProperties as it will never get called in normal operations.
          */
-        String implementingClass = System.getProperty("rxjava.plugin." + classSimpleName + ".implementation");
+        
+        String defaultKey = "rxjava.plugin." + classSimpleName + ".implementation";
```

Comment:
Yes, of course, it's a constant, but I mean explicitly declare it as constant :)